app/views/layouts/application.html.haml

```diff
@@ -0,0 +1,24 @@
+!!!
+%html
+  %head
+    %meta{:content => "text/html; charset=UTF-8", "http-equiv" => "Content-Type"}/
+    %title InstagramClone
+    %link{:href => "https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css", :rel => "styleheet"}/
+    %link{:href => "https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap-theme.min.css", :rel => "stylesheet"}/
```

Comment:
Line is too long. [120/80]<br>Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace